lang/stack/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        commercialhaskell stack 2.1.1
+
+name                stack
+categories          lang haskell
+maintainers         @essandess
+license             BSD
+platforms           darwin
+
+description The Haskell Tool Stack
+long_description    ${description}. \
+        Stack is a cross-platform command line interface for developing Haskell projects. \
+        It features: installing GHC automatically, installing packages needed for your project, \
+        building your project, testing your project, benchmarking your project. \
+        It is intended for Haskellers both new and experienced.
+
+homepage            http://haskellstack.org
+
+variant bootstrap \
+	    description {Bootstrap a build of stack.} {}
+
+master_sites        ${github.homepage}/releases/download/v${github.version}
+distname            ${name}-${github.version}-osx-x86_64
```

Comment:
given that you're getting a file for "x86_64", you probably want to add ```supported_archs     x86_64``` (see the [guide](https://guide.macports.org/#reference.keywords))

Reply:
Yes. Done, thx.

Reply:
Done, thx.